src/ThemeProvider.tsx

```diff
@@ -77,6 +88,11 @@ export function useBootstrapBreakpoints() {
   return breakpoints;
 }
 
+export function useBootstrapMinBreakpoint() {
+  const { minBreakpoint } = useContext(ThemeContext);
+  return minBreakpoint;
+}
+
 export function useIsRTL() {
```

Comment:
Should we maybe throw an exception somewhere in here if the `minBreakpoint` does not exist in the `breakpoints` array?

Reply:
We could probably enforce the ordering of the breakpoints array and have the code check for `breakpoints[0]`.  We could avoid adding the extra prop.   Thoughts?

Reply:
That was actually my first thought too but I didn't want to causing breaking changes between updates for some people. I do like it better though, if you think that's not much of a concern I can change it.

Reply:
Was originally thinking it might not be an issue, but noticed it's been ~3 months since support was released.  I guess this is ok for now, and we'll clean this up in the next major release.  Thanks!